contracts/Conference.sol

```diff
@@ -172,9 +177,50 @@ contract Conference is Destructible, GroupAdmin {
      * @return True if the user is marked as attended by admin.
      */
     function isAttended(address _addr) public view returns (bool){
-        return isRegistered(_addr) && participants[_addr].attended;
+        if (!isRegistered(_addr)) {
+            return false;
+        }
+
+        if (participants[_addr].attended) {
+            return true;
+        }
+
+        // now check the attendance maps
+        if (0 < attendanceMaps.length) {
+            Participant storage p = participants[_addr];
+            uint pIndex = p.participantIndex - 1;
+            uint map = attendanceMaps[uint(pIndex / 256)];
+            return (0 < (map & (2 ** (pIndex % 256))));
+        }
+
+        return false;
```

Comment:
What is the likelihood to reach here? Is it better to throw?

Reply:
It's a getter function so I'd rather not throw.